<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>398</width>
                <height>357</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="diagram">
                            <property name="minimumSize">
                                <size>
                                    <width>378</width>
                                    <height>246</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">QWidget{
                                    border: 2px solid gray;
                                    background: rgb(255, 255, 255)
                                    }
                                </string>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="led_main_chamber">
                                    <property name="minimumSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Main</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="led_load_lock">
                                    <property name="minimumSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Load</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="led_cryo">
                                    <property name="minimumSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Cryo</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="2" column="0">
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QPushButton" name="main_chamber_switch">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>25</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">QPushButton{
                                            background: rgb(193, 193, 193)
                                            }
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>Main Chamber</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="load_lock_switch">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>25</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">QPushButton{
                                            background: rgb(193, 193, 193)
                                            }
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>Load Lock</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="cryo_switch">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>25</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">QPushButton{
                                            background: rgb(193, 193, 193)
                                            }
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>Cryo</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
